src/uct/ib/mlx5/ib_mlx5_log.c

```diff
@@ -86,6 +81,12 @@ ucs_status_t uct_ib_mlx5_completion_with_err(uct_ib_iface_t *iface,
     case MLX5_CQE_SYNDROME_LOCAL_PROT_ERR:
         snprintf(err_info, sizeof(err_info), "Local protection");
         break;
+    case MLX5_CQE_SYNDROME_WR_FLUSH_ERR:
+        snprintf(err_info, sizeof(err_info),
+                 "WR flushed because QP in error state");
+        log_level = UCS_LOG_LEVEL_TRACE;
+        status    = UCS_ERR_CANCELED;
+        break;
```

Comment:
maybe add ucs_log_is_enabled(log_level) in line 127

Reply:
done